studio/src/editors/merch-card-editor.js

```diff
@@ -252,6 +253,16 @@ class MerchCardEditor extends LitElement {
                     >${unsafeHTML(form.ctas.values[0])}</rte-field
                 >
             </sp-field-group>
+            <sp-field-group>
+                <sp-field-label for="osi">OSI</sp-field-label>
```

Comment:
Just "OSI" is not descriptive, I suggest adding "search" i.e. "OSI Search". When I first opened the page, I didn't notice that there is a text field at first.